source/blog/2016-02-24-devconf-2016.html.md

```diff
@@ -0,0 +1,42 @@
+- Attended [Upstream First Testing - Tim Flink](https://www.youtube.com/watch?v=15-yXOJuonQ "Upstream First Testing - Tim Flink"). I discussed with the presenter the possibility to run oVirt with [Lago project on GitHub](https://github.com/lago-project/lago "Lago project on GitHub") in Fedora, in a CI way, by monitoring distgit changes for relevant packages (such as lvm2, device-mapper*, libvirt and others.
+- Attended [High performance VMs in OpenStack - Nikola Dipanov](https://www.youtube.com/watch?v=9J_sEdlpIlQ "High performance VMs in OpenStack - Nikola Dipanov") - looks like oVirt is already doing a lot of what OpenStack is working on to achieve high-performance from KVM!
+- Preached on running Lago with OpenStack, Gluster and Cockpit. At least the first two items look really promising and I'm looking forward to a collaboration in those efforts. I might just try to do the Gluster one myself.
+- [Avocado and Jenkins: Test Automation and CI - Lukáš Doktor, Yash Mankad](https://www.youtube.com/watch?v=rqavfmPAt7o "Avocado and Jenkins: Test Automation and CI - Lukáš Doktor, Yash Mankad") was interesting for me, as Avocado seems like a cool, mature testing framework - which I hope we can use in the future in CI testing.
+- Having used the [oVirt Live](http://www.ovirt.org/download/ovirt-live/"http://www.ovirt.org/download/ovirt-live/") USB DoK image in both FOSDEM and DevConf, I've found several items where we can improve in and filed relevant RFEs for them [1][2].
+
+- The oVirt team has delivered numerous presentations in the virtualization track:
+[Smart VM Scheduling - Martin Sivák](https://www.youtube.com/watch?v=cQqJEiK7-Ug "Smart VM Scheduling - Martin Sivák")
+[Host fencing in oVirt - Fixing the unknown and allowing VMs to be highly available - Martin Peřina](https://www.youtube.com/watch?v=V1JQtmdleaM "Host fencing in oVirt - Fixing the unknown and allowing VMs to be highly available - Martin Peřina") ,
+[Ceph integration with oVirt using Cinder - Nir Soffer](https://www.youtube.com/watch?v=4CbHTAkVDZo "Ceph integration with oVirt using Cinder - Nir Soffer").
+- The Gluster team has additionally provided more presentations:
+[oVirt and Gluster Hyperconvergence - Ramesh Nachimuthu](https://www.youtube.com/watch?v=XudYwEWQF7U "oVirt and Gluster Hyperconvergence - Ramesh Nachimuthu") ,
+[Improvements in gluster for virtualization use case - Prasanna Kumar Kalever](https://www.youtube.com/watch?v=TczVVCbm8NE "Improvements in gluster for virtualization use case - Prasanna Kumar Kalever"), which were all well received.
+
+I'd like to thank our project's community managers Mikey and Brian for once again, just one week after FOSDEM, leading the effort of representing the oVirt project and community in this event.
```

Comment:
"our project's currenty community manager Mikey Ariel, and former community manager Brian Proffitt"